Services/Devices.cs

```diff
@@ -403,6 +519,67 @@ private int AnalyzeBatchErrors(BulkRegistryOperationResult result)
             return errorsByType.Count;
         }
 
+        private async Task<CloudBlockBlob> WriteDevicesToBlobAsync(List<string> serializedDevices)
+        {
+            var sb = new StringBuilder();
+            serializedDevices.ForEach(serializedDevice => sb.AppendLine(serializedDevice));
+
+            // Write to blob
+            var blob = await this.CreateImportExportBlobAsync();
+            using (var stream = await blob.OpenWriteAsync())
+            {
+                byte[] bytes = Encoding.UTF8.GetBytes(sb.ToString());
+                for (var i = 0; i < bytes.Length; i += 500)
+                {
+                    int length = Math.Min(bytes.Length - i, 500);
+                    await stream.WriteAsync(bytes, i, length);
+                }
+            }
+
+            return blob;
+        }
+
+        private async Task<CloudBlockBlob> CreateImportExportBlobAsync()
+        {
+            // Container for the files managed by Azure IoT SDK.
+            // Note: use a new container to speed up the operation and avoid old files left over
+            string containerName = ("iothub-" + DateTimeOffset.UtcNow.ToString("yyyy-MM-dd-HH-mm-ss-") + Guid.NewGuid().ToString("N")).ToLowerInvariant();
```

Comment:
How about adding the IoT Hub name to the containerName? That might help you find the container in Storage Explorer, etc. easier, if we ever need to.

Reply:
it would need extra validation, e.g. I don't know if the chars in the hub name are valid chars in the container name. We can do it separately if finding blobs is too difficult

Reply:
It would be great if all of our resources followed the same naming convention.  When deploying simulation, many had the same prefix and most had the same suffix but not all.  Can we put this in a shared library or construct the prefix/suffix at the beginning of the deployment?